radio/src/Makefile

```diff
@@ -784,6 +784,12 @@ ifeq ($(PCB), $(filter $(PCB), SKY9X 9XRPRO AR9X))
     FLAVOUR = 9xrpro
     CPPDEFS += -DREVX
     LDSCRIPT = targets/sky9x/sam3s8c_flash.ld
+  ifeq ($(NANO), YES)
+    # use newlib-nano for linking
+    NEWLIB_NANO_FLAGS = --specs=nano.specs -u _printf_float
```

Comment:
I suspect this has slipped in when I moved my build config between master and next and back. As I said above, please disregard all changes in the Makefile, except for the one in the last changeset.